packages/amplify-codegen/src/commands/models.js

```diff
@@ -59,13 +59,22 @@ async function generateModels(context) {
     isTimestampFieldsAdded = false;
   }
 
+  //get timestamp config value
+  let emitAuthProvider = false;
+  try {
+    emitAuthProvider = FeatureFlags.getBoolean('codegen.emitAuthProvider');
+  } catch (err) {
+    emitAuthProvider = false;
+  }
+
```

Comment:
This can be refactored to use the `readFeatureFlag` function included in this PR https://github.com/aws-amplify/amplify-codegen/pull/174/files#diff-9cdc392599e83dc816cb33512961b1b923e38faa0692da825ccdaa048081cc6bR21